mindmeld/components/classifier.py

```diff
@@ -19,7 +19,7 @@
 import os
 from abc import ABC, abstractmethod
 
-from sklearn.externals import joblib
+import joblib
```

Comment:
We have to use the older version of joblib due to backwards compatability issues with older models. Is this a required change for Dagshub?

Reply:
Hey @vijay120, no, I don't think it does. My setup was complaining so I changed it (mistakenly thought it would be an improvement), but admittedly I wasn't thinking about backward compatibility issues. Will undo it and commit.

Reply:
Removed all the updated imports, though I must say that when I try to install the package it tells me the specifically allowed versions don't work (using Python 3.8.5, so it might be a python version compatibility issue). If you're aware of this, then that's cool, but thought it might be important to modify the `setup.py` to support newer python versions

Reply:
Got it. We currently support 3.6 and 3.7, we are working on getting the backwards incompatible 3.8 version of MindMeld in soon. Thanks for revert.